src/background/services/openPayments.ts

```diff
@@ -484,6 +504,10 @@ export class OpenPaymentsService {
     return grantDetails;
   }
 
+  private async addPublicKeyToWallet(_walletAddress: WalletAddress) {
+    throw new ErrorWithKey('connectWalletKeyService_error_notImplemented');
```

Comment:
This will be implemented in follow-up PRs. This PR changes the UI and handles the fallback case where we fail to add key for some reason.